components/style/gecko/wrapper.rs

```diff
@@ -319,14 +333,23 @@ impl<'ln> TNode for GeckoNode<'ln> {
         panic!("Atomic child count not implemented in Gecko");
     }
 
-    #[inline(always)]
-    fn borrow_data(&self) -> Option<AtomicRef<PersistentStyleData>> {
-        self.get_node_data().as_ref().map(|d| d.0.borrow())
+    fn get_existing_style(&self) -> Option<Arc<ComputedValues>> {
```

Comment:
You mean in terms of the extra refcount bump? I'm aware of that, but the borrow is pretty hard to preserve in the case where the computed style hangs off the nsIFrame. That's the reason for this refactoring.

If there are other behavior changes I'm not seeing please let me know.